packages/docs-preview/bin/cli.js

```diff
@@ -0,0 +1,75 @@
+const {readFile, watch, writeFile} = require('fs/promises');
+const {spawn} = require('child_process');
+const path = require('path');
+
+const docsMetaFile = path.resolve(process.cwd(), process.argv[2]);
+
+console.log('Watching: ' + docsMetaFile);
+
+async function run() {
```

Comment:
I don't love this code, but it works. Initially I passed an env variable to the remix app with the path to the metafile. Then the remix app could just `fs.readFile` the metafile within a loader. The issue is it didn't pickup changes. It's kinda heavy to copy a whole file into the remix app, but it seemed simplest to just do that have have all of Remix's dev-server file watchers take care of everything.